app/src/main/resources/script.js

```diff
@@ -20,41 +21,89 @@ function clean() {
     }
 }
 
-function createRedirectButton(transferType) {
-    // clean up instances
-    clean();
+function tokenRedirectPath() {
+    var path = '';
+    if (selectedTransferType === 'STANDING_ORDER') {
+        path = '/standing-order?';
+    } else if (selectedTransferType === 'FUTURE_DATED') {
+        path = '/future-dated?';
+    } else if (selectedTransferType === 'ONE_STEP') {
+        path = '/one-step-payment?';
+    } else if (selectedTransferType === 'CROSS_BORDER') {
+        path = '/cross-border?';
+    } else {
+        path = '/transfer?'
+    }
+    return path;
+}
 
-    // Client side Token object for creating the Token button, handling the Token Controller, etc
-    var token = new window.Token({
-        env: 'sandbox',
-    });
+function redeemTokenPopUp() {
+    var path = "";
+    if (selectedTransferType === 'STANDING_ORDER') {
+        path = '/redeem-standing-order-popup';
+    } else if (selectedTransferType === 'FUTURE_DATED') {
+        path = '/redeem-future-dated-popup';
+    } else if (selectedTransferType === 'ONE_STEP') {
+        path = '/redeem-popup';
```

Comment:
same thing here it will be no-op as there won't be any redeem in case of one-step

Reply:
done